projects/angular/src/forms/datepicker/providers/date-navigation.service.ts

```diff
@@ -68,9 +70,11 @@ export class DateNavigationService {
    * Notifies that the selected day has changed so that the date can be emitted to the user.
    * Note: Only to be called from day.ts
    */
-  notifySelectedDayChanged(dayModel: DayModel) {
+  notifySelectedDayChanged(dayModel: DayModel, emitEvent = true) {
```

Comment:
The "Note: Only to be called from day.ts" comment is now incorrect. It should either be removed or updated because this method is now called in different places.